pkg/util/intest/not_in_unittest.go

```diff
@@ -17,4 +17,4 @@
 package intest
 
 // InTest checks if the code is running in test.
-const InTest = false
```

Comment:
if it's not a `const`, maybe in the compile stage golang can't eliminate the check. I'm not sure how much it will affect performance.

we already have `intest` and `enableassert` build tags, can we make use of these build tags?

Reply:
It may not be easy to get these images in TCMS if using build tags.